### PR TITLE
Fix mismatch in column and sort table

### PR DESCRIPTION
**Pull request by shazmasiddiqui**
Changes filter name to 'Title' to match with the column name
PARTIAL Pull Request for Issue https://github.com/joomla/joomla-cms/issues/37499 .

### Before
before

### After
after

### CODE REVIEW

**Comment by ChristineWk** [comment]
I have tested this item :white_check_mark: successfully on 346f733ea34906b10dce20c6076a3917598409a6<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37527">issues.joomla.org/tracker/joomla-cms/37527</a>.</sub>

File: administrator/language/en-GB/com_languages.ini [hunk 1]
```diff
@@ -46,6 +46,8 @@ COM_LANGUAGES_HEADING_LANG_TAG_DESC="Language Tag descending"
 COM_LANGUAGES_HEADING_LANGUAGE="Language"
 COM_LANGUAGES_HEADING_LANGUAGE_ASC="Language ascending"
 COM_LANGUAGES_HEADING_LANGUAGE_DESC="Language descending"
```
Comment (Kostelano) [inline administrator/language/en-GB/com_languages.ini]:
```suggestion
```

**Review by Kostelano — CHANGES_REQUESTED** [review]

File: administrator/language/en-GB/com_languages.ini (hunk 1 above)
Comment (Kostelano) [reply administrator/language/en-GB/com_languages.ini]: These constants are no longer used

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on ecad4ed990f275a404d2cb38b88642395bc694ec<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37527">issues.joomla.org/tracker/joomla-cms/37527</a>.</sub>

**Comment by ChristineWk** [comment]
I have tested this item :white_check_mark: successfully on ecad4ed990f275a404d2cb38b88642395bc694ec<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37527">issues.joomla.org/tracker/joomla-cms/37527</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37527">issues.joomla.org/tracker/joomla-cms/37527</a>.</sub>

**Comment by bembelimen** [comment]
Thx

**Pull request merged**